src/main/java/core/basesyntax/Colours.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colours {
```

Comment:
You have a method `getRandomColor()` (named without `u`). Make everywhere names without `u`.